src/reference.yaml

```diff
@@ -15047,6 +15050,249 @@ paths:
             X-Request-Id:
               type: string
               description: A unique reference for the request.
+  /demo/funding/create:
+    post:
+      tags:
+        - Demo
+      x-api-group: collect
+      summary: Emulate inbound funds
+      description: >-
+        Triggers prod like flow for processing funds with topping up CM balance or rejecting the transaction without topping up CM balance
```

Comment:
Should probably update the description here to indicate it will only be available in Demo and not in Production. Unless we intend to build something to make this clear in the documentation based on the "Demo" tag.